lib/core/notifications/components/notification-history.component.html

```diff
@@ -1,34 +1,73 @@
 <div (keyup)="onKeyPress($event)">
-    <button mat-button [matMenuTriggerFor]="menu" class="adf-notification-history-menu_button"
-            id="adf-notification-history-open-button">
-        <mat-icon>mail_outline</mat-icon>
+    <button mat-button
+            [matMenuTriggerFor]="menu"
+            class="adf-notification-history-menu_button"
+            id="adf-notification-history-open-button"
+            (menuOpened)="onMenuOpened()">
+        <mat-icon matBadge="&#8288;"
```

Comment:
this color should come from the theme

Reply:
This is not a color but a blank space in unicode. We're not showing the total number of notifications but a bubble that appears when you have notifications pending.